src/libstd/ffi/c_str.rs

```diff
@@ -336,6 +377,35 @@ impl From<NulError> for io::Error {
     }
 }
 
+impl IntoStringError {
+    /// Consumes this error, returning original `CString` which generated the
+    /// error.
+    #[unstable(feature = "convert", reason = "recently added", issue = "0")]
+    pub fn into_inner(self) -> CString {
```

Comment:
Yeah let's call this `into_cstring` for now to stick with `into_<type>` of the other errors.